docs/pages/fund-agents-apps/fund-your-app.md

```diff
@@ -0,0 +1,201 @@
+
+The Funding Portal UI displays a warning if your allowance goes below XXXX does it do thisXXXX.
+
+If your allowance goes to zero, the Payer Registry smart contract rejects new messages sent to the XMTP Broadcast Network and returns an `INSUFFICIENT_PAYER_BALANCE` error.
+
+### Step 6: Withdraw and claim funds
+
+You can use the payer wallet (and only the payer wallet) to withdraw USDC from its messaging fee allowance at any time.
+
+- Withdrawals that require bridging will be available after 7 days and require a second transaction to claim. 
+- Withdrawals without bridging will be available after 48 hours and require a second transaction to claim.
+
+1. Connect your payer wallet to the XMTP Funding Portal: [https://fund.xmtp.org/](http://fund.xmtp.org/).
+2. On the **Dashboard** page, click **Withdraw**.
+3. Enter the amount of USDC you want to withdraw from your messaging balance. Click **MAX** if you want to withdraw the maximum amount available.
+4. Click **Request withdrawal**.
+5. XXXX Need to wait to see this part of the UI XXXXX.
+6. After 48 hours, return to the XMTP Funding Portal to complete your withdrawal. On the homepage, view the **Transaction** panel and locate your **Withdrawal** transaction. The **Status** column should be set to **_Ready to withdraw_**.
+7. Click **Ready to withdraw** to display the **Transaction details** panel.
+8. Verify the withdrawal details and click **Claim USDC**.
+9. XXXX Need to wait to see this part of the UI XXXXX.
+
+## Fund an app using a wallet other than its payer wallet
+
+### Step 1. Get the payer wallet address
+
+To fund an app using a wallet other than its payer wallet, you need the payer wallet address.
+
+### Step 2. Register the payer wallet
+
+1. Use the wallet you want to use to fund the app to connect to the XMTP Funding Portal: [https://fund.xmtp.org/](http://fund.xmtp.org/).
+2. On the **Welcome** page, click **Use other wallet**.
+3. On the **Manage payer wallets** screen, enter the payer wallet address and a human-readable display name.
+4. Click the green check button to save.
+
+Your connected wallet can now fund the registered payer wallet.
+
+### Step 3: Deposit USDC into your wallet
+
+Deposit USDC into your wallet on Base. This is the wallet you want to use to fund the payer wallet.
+
+Here are some sources you can use to acquire USDC:
+
+- Centralized exchanges: Binance, Coinbase, Kraken
+- Direct purchase: Circle, Coinbase
+- Business accounts: Circle business accounts for larger operations
+
+To get USDC on Base Sepolia, you can use [https://faucet.circle.com/](https://faucet.circle.com/), which provides 10 USDC per hour. 
+
+### Step 4: Allocate funds for messaging
+
+The XMTP Funding Portal will accept only testnet USDC until **November 1, 2025**. Plan your testnet and mainnet funding accordingly.
+
+1. Use the wallet you want to use to fund the app to connect to the XMTP Funding Portal: [https://fund.xmtp.org/](http://fund.xmtp.org/).
```

Comment:
```suggestion
1. Use the wallet you funded in Step 3 to connect to the XMTP Funding Portal: [https://fund.xmtp.org/](http://fund.xmtp.org/).
```

or 

```suggestion
1. Use your wallet to connect to the XMTP Funding Portal: [https://fund.xmtp.org/](http://fund.xmtp.org/).
```

making these suggestions so it's clear we're not introducing a third wallet